types/types.go

```diff
@@ -309,3 +334,37 @@ type AccessLog struct {
 	FilePath string `json:"file,omitempty" description:"Access log file path"`
 	Format   string `json:"format,omitempty" description:"Access log format: json | common"`
 }
+
+// HasCustomHeadersDefined checks to see if any of the custom header elements have been set
+func (h *Headers) HasCustomHeadersDefined() bool {
+	if len(h.CustomResponseHeaders) == 0 &&
+		len(h.CustomRequestHeaders) == 0 {
+		return false
+	}
+	return true
+}
+
+// HasSecureHeadersDefined checks to see if any of the secure header elements have been set
+func (h *Headers) HasSecureHeadersDefined() bool {
```

Comment:
This method doesn't work. Bool are initiate to false not to true 😉 
Unit tests are your friends:

**types_test.go**
```golang
package types

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func TestHeaders_ShouldReturnFalseWhenNotHasCustomHeadersDefined(t *testing.T) {
	headers := Headers{}

	assert.False(t, headers.HasCustomHeadersDefined())
}

func TestHeaders_ShouldReturnTrueWhenHasCustomHeadersDefined(t *testing.T) {
	headers := Headers{}

	headers.CustomRequestHeaders = make(map[string]string)
	headers.CustomRequestHeaders["foo"] = "bar"

	assert.True(t, headers.HasCustomHeadersDefined())
}

func TestHeaders_ShouldReturnFalseWhenNotHasSecureHeadersDefined(t *testing.T) {
	headers := Headers{}

	assert.False(t, headers.HasSecureHeadersDefined())
}

func TestHeaders_ShouldReturnTrueWhenHasSecureHeadersDefined(t *testing.T) {
	headers := Headers{}

	headers.SSLRedirect = true

	assert.True(t, headers.HasSecureHeadersDefined())
}

```